### DISCUSSION TITLE
3 Phase suport.

### DISCUSSION THREAD

**Discussion by virus100b** [jello1974/duosidaEV-home-assistant#9] [General]
Hello,

I understand that the charger come equipped with 3x6mm cable for 32A, 7.4Kw.
I have an WV eUp that support 7.4 kw charging but in a configuration of 2 x16A using 2 phase. So i will  be able to charge my car with this station just with 3.2kw.
Has anyone an idea if the charging cable going to the car socket is dressed with wires for the other 2  phases?
Some picture with  PCB layout will help.

Thank you!

**Comment by jello1974** [comment]
never heard of 2 phase chargers...
What type of connector your car has? Check this.
https://wallbox.com/en_catalog/faqs-plug-types
and this
https://evbox.com/en/electric-cars/volkswagen/volkswagen-e-up

**Comment by virus100b** [comment]
The charger is 3 phase Type 2. My car has just 2 chargers onboard and can charge 2 x 16A. total of 7.2kW
![eUp charge](https://github.com/jello1974/duosidaEV-home-assistant/assets/64686560/2d8060ca-f0b1-4110-9c85-860371be5464)

**Comment by jello1974** [comment]
You need to search then for a 3-Phase charger. The one i have is 1-Phase.

**Comment by spainpiio** [comment]
I just installed the 3 phase 22kW Wallbox and the integration works fine.
Thank You very much!
